### ISSUE TITLE
Abnormal circulating Myoglobin concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6002]
**Preferred term label:** Abnormal circulating Myoglobin concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of myoglobin in the blood is outside the normal range., PMID: 15339940

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased circulating Myoglobin concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of myoglobin in the blood is below the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Myoglobin concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Increased circulating Myoglobin concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of myoglobin in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Myoglobin concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
There is no medically relevant "too low" circulating myoglobin, only trace amounts are found in the circulation
https://www.ncbi.nlm.nih.gov/books/NBK470441/

**Comment by pnrobinson** [comment]
added
